utils/src/main/java/io/neow3j/io/BinaryReader.java

```diff
@@ -274,7 +289,7 @@ public short readShort() throws IOException {
     }
 
     public byte[] readVarBytes() throws IOException {
-        return readVarBytes(0x7fffffc7);
+        return readVarBytes(0x1000000);
```

Comment:
Was this wrong before? Or things changed for Neo3?

Reply:
Around 16MB (0x1000000) is the max used in neo-core, though, it is not a recent change in their code. It has been like that since 2018.